components/ChangePasswordForm/index.js

```diff
@@ -22,18 +24,26 @@ const ChangePasswordForm = () => {
         placeholder="Confirm New Password"
       />
       <div className={styles.signInButtonPair}>
-        <input
-          className={styles.signInButtons}
-          type="button"
-          name="cancelPassword"
-          value="Cancel"
-        />
-        <input
-          className={styles.signInButtons}
-          type="submit"
-          name="savePassword"
-          value="Save"
-        />
+        <Link href="/settings">
```

Comment:
Nice work putting this together!

It's invalid HTML to nest an input into an anchor tag. This is because both handle click events.

When it's time to wire this up, let's review the desired UX and we can come up with a valid HTML solution.